README.md

```diff
@@ -71,15 +71,12 @@ use 'f4z3r/gruvbox-material.nvim'
 
 
 ```lua
-
 {
   'f4z3r/gruvbox-material.nvim',
-  name = 'gruvbox-material',
-  lazy = false,
-  priority = 1000,
-  config = function()
-    -- configs go here
-  end,
+    name = 'gruvbox-material',
+	  lazy = false,
+	  priority = 1000,
+	  opts = {},
```

Comment:
The spacing is still strange here. There seems to be a space too many, and tabs combined ...